### PR TITLE
USHIFT-1774: Add router stress tests

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: assets/components/openshift-router/cluster-role-binding.yaml [hunk 1]
```diff
@@ -9,5 +9,5 @@ subjects:
   namespace: openshift-ingress
 roleRef:
   kind: ClusterRole
-  name: openshift-ingress-router
+  name: system:router
```
Comment (dhellmann) [inline assets/components/openshift-router/cluster-role-binding.yaml:12]: I commented on another PR about renaming these.

**Comment by pacevedom** [comment]
/hol.d

**Pull request merged**